README.md

```diff
@@ -4,37 +4,61 @@
 
 This repository contains the assets required to build the [Kubernetes website and documentation](https://kubernetes.io/). We're glad that you want to contribute!
 
-## Running the website locally using Hugo
+# Using this repository
 
-See the [official Hugo documentation](https://gohugo.io/getting-started/installing/) for Hugo installation instructions. Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
+You can run the website locally using Hugo, or you can run it in a container runtime. We strongly recommend using the container runtime, as it gives deployment consistency with the live website.
 
-Before building the site, clone the Kubernetes website repository:
+## Prerequisites
 
-```bash
+To use this repository, you need the following installed locally:
+
+- [yarn](https://yarnpkg.com/)
+- [npm](https://www.npmjs.com/)
```

Comment:
I hope this is the same as the documentation in the site's `contribute` section.

Reply:
I would recommend the `minimal` installation requirements.

Reply:
Just looked at the "Learn to work from a local clone of the repository." section and the link breaks: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#fork-the-repo

I will open an issue.

Reply:
https://github.com/kubernetes/website/issues/22607